src/BlazorUI/Bit.BlazorUI/Utils/Theme/bit-theme.ts

```
@@ -35,6 +40,7 @@ class BitTheme {
 
     public static set(themeName: string) {
         this.currentTheme = themeName;
+        localStorage.setItem(this.THEME_SOTRAGE_KEY, themeName);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)